app/views/deals/_deal.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="m-2 bg-white border d-flex justify-content-between p-2" id="<%= dom_id deal %>">
+  <p>
+    <%= deal.name %>
+  </p>
+
+  <p>
+    <%= deal.amount %>
+  </p>
+
```

Comment:
- [ ] In this page, Please show the date when the transaction happens as per the design.

### Your design
![Screenshot from 2022-04-29 23-57-50](https://user-images.githubusercontent.com/22434660/166008315-7536f567-6aed-43ba-95f0-b81cda43294d.png)


### GIven design
![Screenshot from 2022-04-30 00-16-20](https://user-images.githubusercontent.com/22434660/166008352-3516ca31-ee95-498f-b454-cb0b444830c2.png)